go/staking/api/sanity_check.go

```diff
@@ -73,21 +73,21 @@ func SanityCheckDelegations(account *Account, delegations map[signature.PublicKe
 	sharesExpected := account.Escrow.Active.TotalShares
 
 	if shares.Cmp(&sharesExpected) != 0 {
-		return fmt.Errorf("staking: sanity check failed: all shares of all delegations for account don't add up to account's total active shares in escrow")
+		return fmt.Errorf("staking: sanity check failed: all shares of all delegations (%s) for account with ID: %s don't add up to account's total active shares in escrow (%s)", shares, id, sharesExpected)
```

Comment:
:information_source: in some cases, error diagnostics is the only thing that uses the `id` parameter. but adding the argument also makes these sanity check functions uniform with ones that already had an `id` argument.